src/util/findLast.test.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import {findLast, findLastIndex} from "./findLast";
+
+describe("util/findLast", () => {
```

Comment:
Would be nice to test out the empty case here as well (empty array)

Reply:
Accepted and added.